app/controllers/api/v1/media_controller.rb

```diff
@@ -34,8 +34,8 @@ def show
   def update
     error_unless_exists
     set_controlled_resources
-    send_aggregation_ready_email
     super
+    send_aggregation_ready_email
```

Comment:
should be getting called, the spec tests that the mailer gets called with the correct media resource id. I wanted to add it after to ensure the .save! is called on the update action so failed updates won't trigger this.